src/ILCompiler.Compiler/src/Compiler/Compilation.cs

```diff
@@ -106,18 +106,7 @@ void ICompilation.Compile(string outputFile)
             // TODO: Hacky static field
 
             string systemModuleName = ((IAssemblyDesc)NodeFactory.TypeSystemContext.SystemModule).GetName().Name;
-
-            // TODO: CompilationUnitPrefix is used even before this point!!!
-            // TODO: just something to get Runtime.Base compiled
-            if (systemModuleName != "System.Private.CoreLib")
-            {
-                NodeFactory.CompilationUnitPrefix = systemModuleName.Replace(".", "_");
-            }
-            else
-            {
-                NodeFactory.CompilationUnitPrefix = NameMangler.SanitizeName(Path.GetFileNameWithoutExtension(outputFile));
-            }
-
+            
```

Comment:
ach, sometimes the GH client tries to be clever and doesn't include all lines in the file in the commit. Removed.